toolchain/check/import_ref.cpp

```diff
@@ -67,9 +67,7 @@ auto AddImportRef(Context& context, SemIR::ImportIRInst import_ir_inst,
   // somewhere, because it's referenced by other instructions and needs to be
   // visible in textual IR. Adding it to the file block is arbitrary but is the
   // best place we have right now.
```

Comment:
Should we also remove the last line of this comment?

Reply:
The comment in context.h was copying the important bits, I'd meant to remove this. So I've cut the detail out since it's now on the vector.